Chapter1/4-Shaders-MoreAttributes/Window.cs

```diff
@@ -7,12 +7,11 @@
 
 namespace LearnOpenTK
 {
-    // In this project we will be assigning 3 colors to the triangle, one for vertex, but the output will be a mixture
-    // Of the three depending on how far away from the assignment point the color pixel is
-    // This comes down to something called fragment interpolation in the fragment shader
-    // When rendering a triangle the rasterization stage usually results in a lot more fragments than vertices originally specified. 
-    // The rasterizer then determines the positions of each of those fragments based on where they reside on the triangle shape
-    // Based on these positions, it interpolates all the fragment shader's input variables
+    // In this project we will be assigning 3 colors to the triangle, 
```

Comment:
Your first comment here made more sense, let me write an idea to what you could write here.
```
In this project we will be assigning 3 colors to the triangle, one for vertex.
The output will be an interpolated value based on the distance from each vertex.
If you want to look more into it the in-between step is called a Rasterizer.
```
Trying to keep it short and simple while correct. Also giving a way to further look into the topic if anyone is interested. Ofc feel free to change it as you like.